swiggy-pdp-cart/src/components/Main/Content/Menu/Menu.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Dish from "./Dish/Dish";
+import MenuCSS from './menu.module.css'
+export default function Menu({menuHeading,menuList}) {
+  return (
+    <div className={MenuCSS.menu+' col-4'}>
+      <h2>{menuHeading}</h2>
+      <p>{menuList.length} ITEMS</p>
+      <ul>
+        {menuList.map((dish) => {
+          const {id,displayName,price}=dish;
```

Comment:
Always code in a declarative manner as opposed to imperative manner.

Instead of having an inline function here, move this to a named function `renderMenuListItem`